client/scss/components/_chooser.scss

```diff
@@ -64,6 +64,21 @@
   @apply w-body-text-large;
 }
 
+@media (hover: hover) {
+  .chooser__actions {
+    opacity: 0;
+  }
+
+  .w-field__input:hover,
+  .w-field__input:focus-within {
```

Comment:
This seems problematic to me, as it means that if I hover/focus within a StreamField, all choosers inside that StreamField will get displayed.

We’d either need to change this back to `.chooser:hover, .chooser:focus-within`, or add a descendant selector (which I was trying to avoid as it’s a bit brittle to do this across different components).